content/blog/2024-07-02-tmix-06/index.md

```diff
@@ -0,0 +1,66 @@
++++
+title = "This Month in Xilem, June 2024"
+authors = ["Daniel McNab"]
++++
+
+We released [Vello 0.2.0][] at the start of the month, which included some key improvements discussed in previous months' updates.
+In June, we merged initial handling of Emoji ([vello#615][]), which supports glyphs which use the COLR specification (Apple's system emoji font does not use this format and so currently will not render).
```

Comment:
Mentioning "In June" is odd in the context of a June update. It makes you wonder if this isn't a June update.